### PR TITLE
fix(docs): update testing documentation

### LINKED ISSUE #711: Update testing documentation
Update [this doc](https://github.com/GoogleCloudPlatform/emblem/blob/main/docs/testing.md), including **fixing broken links**.

### PR DESCRIPTION
**Pull request by ace-n**
Fixes #711

### CODE REVIEW

File: docs/testing.md [hunk 1]
```diff
@@ -4,29 +4,23 @@ These tests run automatically once you have set up a working instance of Emblem.
...
-The **delivery system** can _optionally_ be covered by an automatic **End-to-End (E2E) Test**.
-
-This test uses Cloud Build to run the [setup/deployment script](/setup.sh) on a nightly basis, and checks that it completes without any errors.
-  
-Follow the [instructions](/installation-testing/README.md) in the `installation-testing` directory to enable this test.
+Content API unit tests run automatically [on pull requests to `main`](/terraform/modules/ops/testing.tf#:~:text=resource%20%22google_cloudbuild_trigger%22%20%22api_unit_tests%22).
 
 ## Website
 
 The [Website](/docs/website.md) component is covered by automatic **End-to-End (E2E) Tests**.
```
Comment (rogerthatdev) [inline docs/testing.md:11]: nit: capitalization

**end-to-end (e2e) tests**

Comment (rogerthatdev) [inline docs/testing.md:11]: nit: capitalization 
[website]

File: docs/testing.md [hunk 2]
```diff
@@ -4,29 +4,23 @@ These tests run automatically once you have set up a working instance of Emblem.
...
   
 These tests run the website in a [Docker container](/ops/e2e-runner), and check that the website's routes load without any errors.
 
-These tests run automatically under two circumstances:
-* [nightly against `main`](/terraform/ops/testing.tf#:~:text=e2e_nightly_tests)
-* [when the `main` branch is updated](/terraform/modules/ops/build.tf#:~:text=e2e_testing_build_runner).
+> **Note:** this Docker container is rebuilt [nightly](/terraform/modules/website-e2e-test/build.tf#:~:text=testing_web_e2e_build_container_trigger).
+>
+> We kept nightly {container builds, test runs} separate so that E2E tests run even if the container build fails.
```
Comment (rogerthatdev) [inline docs/testing.md:17]: nit: tense

keep

Comment (rogerthatdev) [inline docs/testing.md:17]: replace bracket usage with plain English

container builds and test runs

File: docs/testing.md (hunk 1 above)
Comment (ace-n) [reply docs/testing.md]: Done.

File: docs/testing.md [hunk 3]
```diff
@@ -4,29 +4,23 @@ These tests run automatically once you have set up a working instance of Emblem.
...
-This test uses Cloud Build to run the [setup/deployment script](/setup.sh) on a nightly basis, and checks that it completes without any errors.
-  
-Follow the [instructions](/installation-testing/README.md) in the `installation-testing` directory to enable this test.
+Content API unit tests run automatically [on pull requests to `main`](/terraform/modules/ops/testing.tf#:~:text=resource%20%22google_cloudbuild_trigger%22%20%22api_unit_tests%22).
 
 ## Website
 
 The [Website](/docs/website.md) component is covered by automatic **End-to-End (E2E) Tests**.
   
 These tests run the website in a [Docker container](/ops/e2e-runner), and check that the website's routes load without any errors.
```
[the remark this answers is not in the document]
Comment (ace-n) [reply docs/testing.md]: Done.

File: docs/testing.md (hunk 2 above)
Comment (ace-n) [reply docs/testing.md]: Done.

File: docs/testing.md [hunk 4]
```diff
@@ -4,29 +4,23 @@ These tests run automatically once you have set up a working instance of Emblem.
...
+> **Note:** this Docker container is rebuilt [nightly](/terraform/modules/website-e2e-test/build.tf#:~:text=testing_web_e2e_build_container_trigger).
+>
+> We kept nightly {container builds, test runs} separate so that E2E tests run even if the container build fails.
+
+
+These tests run automatically [on pull requests to `main`](/terraform/modules/website-e2e-test/testing.tf#:~:text=testing_web_e2e_run_tests_trigger).
+
 
-> **Note:** we are working on adding a pull-request runner as well
+> **Note:** we are working on adding a nightly test runner as well
```
Comment (rogerthatdev) [inline docs/testing.md:23]: nit: forecasting features

remove and add when we have this

File: docs/testing.md [hunk 5]
```diff
@@ -92,5 +86,5 @@ gcloud builds submit --tag \
 
 ## Terraform Testing
```
Comment (rogerthatdev) [inline docs/testing.md:87]: remove: there is no terraform testing presently

File: docs/testing.md (hunk 2 above)
Comment (ace-n) [reply docs/testing.md]: Done.

File: docs/testing.md (hunk 4 above)
Comment (ace-n) [reply docs/testing.md:23]: Removed.

File: docs/testing.md (hunk 5 above)
Comment (ace-n) [reply docs/testing.md:87]: Removed.

**Review by rogerthatdev — APPROVED** [review]

**Pull request merged**